backend/src/schema/resolvers/shout.spec.js

```diff
@@ -100,6 +100,18 @@ describe('shout and unshout posts', () => {
           data: { Post: [{ id: 'another-user-post-id', shoutedBy: [{ id: 'current-user-id' }] }] },
           errors: undefined,
         })
+        // Test to make sure SHOUT relationship has "createdAt" date.
+        const relation = await instance.cypher(
+          'MATCH (user:User {id: $userId1})-[relationship:SHOUTED]->(node {id: $userId2}) WHERE relationship.createdAt IS NOT NULL RETURN relationship',
+          {
+            userId1: 'current-user-id',
+            userId2: 'another-user-post-id',
+          },
+        )
+        const relationshipProperties = relation.records.map(
+          record => record.get('relationship').properties.createdAt,
+        )
+        expect(relationshipProperties[0]).toEqual(expect.any(String))
```

Comment:
Same as above :point_up: in this test case we already have two assertions - because of a weakness in the `shout` resolver: Instead of returning the shouted post, we return `Boolean`. That's why we have to send another graphql query just to make sure it really got shouted by the current user.

Reply:
Thanks @roschaefer for review. I made changes as requested.